app/assets/javascripts/lunch.js

```diff
@@ -24,25 +24,41 @@ document.addEventListener('turbolinks:load', function() {
 
   // どのメンバーを表示しないか
   function noDisplayMember(members){
-    const selectedRows = document.querySelectorAll('.selected-row');
-    const selected_projects = Array
-      .from(selectedRows)
-      .map(row => row.children[1].textContent.split(','))
-      .flat()
-      .filter(n => n !== '');
-
     for(const member of members) {
       member.classList.remove('unselectable-row');
-      const arr = member.children[1].textContent.split(',');
-      if (intersection(arr, selected_projects).size !== 0)
+      if ( hasSameProjects(member) || isGone(member))
```

Comment:
```suggestion
      if (hasSameProjects(member) || isGone(member))
```

あまり、こういう空白やフォーマットについての指摘はしたくないので、jslint なりをいれてチェックできるようにするとよいと思います。